client/ayon_hiero/api/pipeline.py

```diff
@@ -35,42 +51,80 @@
 AVALON_CONTAINERS = ":AVALON_CONTAINERS"
 
 
-def install():
-    """Installing Hiero integration."""
 
-    # adding all events
-    events.register_events()
+class HieroHost(
+    HostBase, IWorkfileHost, ILoadHost, IPublishHost
+):
+    name = "hiero"
 
-    log.info("Registering Hiero plug-ins..")
-    pyblish.register_host("hiero")
-    pyblish.register_plugin_path(PUBLISH_PATH)
-    register_loader_plugin_path(LOAD_PATH)
-    register_creator_plugin_path(CREATE_PATH)
+    def open_workfile(self, filepath):
+        return open_file(filepath)
 
-    # register callback for switching publishable
-    pyblish.register_callback("instanceToggled", on_pyblish_instance_toggled)
+    def save_workfile(self, filepath=None):
+        return save_file(filepath)
 
-    # install menu
-    menu.menu_install()
-    menu.add_scripts_menu()
+    def work_root(self, session):
+        return work_root(session)
```

Comment:
I believe this and what is in `workio.py` is essentially the default behavior so we could just remove the override here and remove it from `workio.py`

See parent implementation [here](https://github.com/ynput/ayon-core/blob/da78fbceff3744b3293d0b21b3766641b7d61400/client/ayon_core/host/interfaces.py#L235). However, this is likely better left for a separate PR.